app/models/cohort.rb

```diff
@@ -31,6 +32,10 @@ def self.current
     starting_within(Date.current - 1.year + 1.day, Date.current)
   end
 
+  def self.destination_from_frozen_cohort
+    find_by!(start_year: DESTINATION_START_YEAR_FROM_A_FROZEN_COHORT)
```

Comment:
we have a similar constant here `ParticipantProfile::ECF::DESTINATION_COHORT_WHEN_MOVING_PARTICIPANTS_TO_FROM_A_FROZEN_COHORT` maybe we can use that instead of defining another one in cohort?